cartoframes/assets/src/widgets.js

```diff
@@ -40,23 +40,21 @@ export function bridgeLayerWidgets(map, mapLayer, mapSource, widgets) {
     map: map
   });
 
-  mapLayer.on('loaded', () => {
-    widgets
-      .filter((widget) => widget.has_bridge)
-      .forEach((widget) => renderBridge(bridge, widget, mapLayer));
+  widgets
```

Comment:
Don't we have to wait for map initialization?

Reply:
They're reloaded once the map is updated, when waiting for the initialization it wasn't rendered since the beginning and therefore there was an error.